content/tracing/services/_index.md

```diff
@@ -0,0 +1,106 @@
+---
+title: Services list
+kind: Documentation
+---
+
+## Overview
+
+After having [instrumented your application](/tracing/languages), your reporting services are listed on [the APM services page](https://app.datadoghq.com/apm/services).  
+Select a service to have more performance insights, [read our dedicated service documentation to learn more](tracing/services/service)
+
+{{< img src="tracing/services/services_page.png" alt="Services page" responsive="true" popup="true">}}
+
+## Filtering the service list
+
+The services list is a bird eyed view of all [services](/tracing/services/service) reporting from your infrastructure. It can be filtered depending on:
+
+* [Your environment](#environment).
```

Comment:
Remove period or add to the other two bullets - do we have a style guide for this?

Reply:
👍  https://github.com/DataDog/documentation/blob/master/CONTRIBUTING.md